addons/event/models/event_mail.py

```diff
@@ -190,13 +173,51 @@ def execute(self):
                 })
         return True
 
+    def _execute_after_subscription(self, registration_limit=1000):
```

Comment:
Feels like param "registration_limit" is just there for the test, which is not great.

Wouldn't it be better to have a class property "_AFTER_SUB_BATCH_SIZE", set to 1000, that you can mock in the test?